index.html

```diff
@@ -670,6 +670,25 @@ <h3>
 ]
 </code></pre>
 <p>You can open the RabbitMQ's management console and find these messages in a queue.</p>
+
+<h2>Logging</h2>
+
+
+<p>REMReM publish application logging is implemented using the logback-classic, and we required to configure the logback.xml file, <br/>
```

Comment:
Remove comma at the end of line and add dot if it is needed.
REMReM Publish instead of REMReM publish like in generate documentation